### PR TITLE
add font smoothing mixin

### PR DESCRIPTION
**Pull request by hippogriffic**
@incuna/frontend pls

slightly modified version of libby's mixin from the brown site

### CODE REVIEW

**Comment by pandalion** [comment]
Nice!

File: incuna-sass/mixins/_font-smoothing.sass [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+// To set font-smoothing to auto, or overwrite antialised
+//     .thing
+//         @include font-smoothing(auto)
+
+@mixin font-smoothing($type: antialiased)
+    @if $type == antialiased
+        -moz-osx-font-smoothing: grayscale
+        -webkit-font-smoothing: antialiased
+    @else
```
Comment (maciejpi) [inline incuna-sass/mixins/_font-smoothing.sass]: shouldn't it be `@else $type == auto`?

Comment (henrahmagix) [inline incuna-sass/mixins/_font-smoothing.sass]: I agree: the mixin shouldn't apply any styles if given a bogus argument. A simple typo will result in the opposite effect:
```sass
@include font-smoothing(antaliased)
```
```css
-moz-osx-font-smoothing: auto
-webkit-font-smoothing: auto
```

Comment (henrahmagix) [inline incuna-sass/mixins/_font-smoothing.sass]: Related to above, can you warn the user if they're passing an unknown argument?
```sass
@mixin font-smoothing($type: antialiased)
    @if $type == antialiased
        -moz-osx-font-smoothing: grayscale
        -webkit-font-smoothing: antialiased
    @else if $type == auto
        -moz-osx-font-smoothing: auto
        -webkit-font-smoothing: auto
    @else
        @warn "Unknown argument: #{$type}"
```

Then
```sass
.thing
    @include font-smoothing(crazy)
```
outputs
```css
WARNING: Unknown argument: crazy
Backtrace:
	test.sass:9, in mixin `font-smoothing`
	test.sass:12
```

**Pull request merged**